google-cloud-trace/lib/google/cloud/trace/rails.rb

```diff
@@ -168,6 +169,9 @@ def self.merge_rails_config rails_config
             end
             config.sampler ||= trace_config.sampler
             config.span_id_generator ||= trace_config.span_id_generator
+            if trace_config.default_labels.present?
+              config.default_labels = trace_config.default_labels
```

Comment:
Can we use `||=` so that the direct client config takes precedence over Rails config (for consistency with all the other configs here—note that they all use `||=` or equivalent).

Reply:
`||=` only works on `nil`, but the default here is an empty Hash.

We could introduce a safeguard into the `set_default_labels` method that verifies we're not dealing with `nil`. I find the current solution more elegant, however, cause it pushes that logic to the boundary.

Let me know what you think!